README.md

```diff
@@ -0,0 +1,3 @@
+# robotont\_navigation
+
+This repository is a ROS package that contains configuration files and a launch file for the navigation stack.
```

Comment:
i know there's this slang "navigation stack" but it is obsolete. use "ROS Navigation", instead.

Reply:
put here also travis-ci.**com** status/license